concord/concord.py

```diff
@@ -4,75 +4,50 @@
 from nltk.corpus import stopwords
 from nltk.stem import WordNetLemmatizer
 from sklearn.datasets import fetch_20newsgroups
-
-from bert.bert import initialize_model
+from bert.bert import fit_and_evaluate_model
 
 
 def preprocess_documents(documents):
-    """
-    Preprocess the documents by:
-    - Lowercasing
-    - Removing punctuation
-    - Removing stop words
-    - Lemmatizing
-    """
+    nltk.download('stopwords')
+    nltk.download('punkt')
+    nltk.download('wordnet')
```

Comment:
Manque un appel à nltk.download('punkt_tab') pour pouvoir utiiser wordnet

Reply:
This should not be there, this downloads wordlist basically, and I don't want our code to automatically and without user supervision download stuff on the user/dev computer, futhermore calling download blindly will setup the wordlist in the user home which is not great.
This should be handle by the developer directly and just provided to Concord by env variable like wrote in the readme file